### ISSUE TITLE
training with flash-attn

### ISSUE THREAD

**Issue by clf28** [ML-GSAI/LLaDA-V#22]
Hi, 
thanks for you brilliant work! I notice in both pretrain.sh and sft.sh
https://github.com/ML-GSAI/LLaDA-V/blob/main/train/scripts/llada_v_pretrain.sh#L70
https://github.com/ML-GSAI/LLaDA-V/blob/main/train/scripts/train_ablation/llada_v_sft.sh#L82C7-L82C27

, `attn_implementation` is set to sdpa, can it be replaced by flash-atth to accelerate training speed?

**Comment by yyyouy** [comment]
I haven't tried it yet, but you're welcome to give it a try and observe how the loss changes.
